src/plugins/pronoundb/pronoundbUtils.ts

```diff
@@ -150,14 +151,21 @@ async function bulkFetchPronouns(ids: string[]): Promise<PronounsResponse> {
 }
 
 export function formatPronouns(pronouns: string): string {
+    let formattedPronouns;
```

Comment:
```suggestion
    let formattedPronouns: string;
```
or better yet, use a ternary

Reply:
I applied the suggested change. Ternary seems like it would make the line too long, or split into multiple lines which is generally discouraged for a ternary, so idk.